docker/pytmv1/Pipfile

```diff
@@ -9,4 +9,4 @@ pytmv1 = "*"
 [dev-packages]
 
 [requires]
-python_version = "3.11"
\ No newline at end of file
+python_version = "3.11"
```

Comment:
What is the change here?

Reply:
In this section I saw python version "3.11.x" so I removed it to keep it consistent with what was there before. No other changes were made.